docs/usage.md

```diff
@@ -70,6 +70,9 @@ if TYPE_CHECKING:
     import mypy_boto3_s3
 ```
 
+There is some support for imports created with `importlib`: namely any usage with a string literal
```

Comment:
Can we add a small example here, to make it more clear to users which cases are supported and which are not? e.g. something like

---

Imports like this are recognized by deptry:

```py
importlib.import_module("bar")
```

Imports like this are not recognized.

```py
foo = "bar"
importlib.import_module(foo)
```

Reply:
Completed in 9008d49